services/backend/src/tmp/connectivity_pa_timeline_prot_conn.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This is how i think the response should be, but the other mockup is returning an array